### PR TITLE
Deprecate Mean and MeanData

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-java/issues/1094

/cc @dinooliva

### CODE REVIEW

File: contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java [hunk 1]
```diff
@@ -99,7 +98,7 @@
               4096.0, 8192.0, 16384.0, 32768.0, 65536.0));
 
   // Use Aggregation.Mean to record sum and count stats at the same time.
-  @VisibleForTesting static final Aggregation MEAN = Mean.create();
+  @VisibleForTesting static final Aggregation MEAN = Aggregation.Mean.create();
```
Comment (sebright) [inline contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java]: How will we handle the existing Mean RPC views?

Comment (songy23) [reply contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java]: Mean is only used in interval views, and those views are not exported to any backends. I think we can remove those views along with Mean and Window in the future.

Comment (sebright) [reply contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java]: The error count views also use Mean, though I think they also need to be updated to use different Measures, so we can update the Aggregation type at the same time.

Comment (songy23) [reply contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java]: Oh correct, the error count views should probably be updated to use Count instead.

**Comment by songy23** [comment]
Looks like AppVeyor is having some difficulties connecting to maven. I saw another AppVeyor failure in another PR. I'll do a dummy push to run AppVeyor test again later today.

**Comment by songy23** [comment]
Finally got CI working...

**Pull request merged**